tsconfig.json

```diff
@@ -1,16 +1,18 @@
 {
-    "compilerOptions": {
-        "rootDir": ".",
-        "module": "commonjs",
-        "target": "es6",
-        "outDir": "out",
-        "lib": [
-            "es2017",
-            "DOM"
-        ],
-        "sourceMap": true
-    },
-    "exclude": [
-        "node_modules"
-    ]
+  "compilerOptions": {
+    // NOTE: The TypeScript compiler is only used for building the tests (and
+    // the sources which the tests need). The extension is built with `esbuild`.
```

Comment:
Does this file configure the typescript compiler rather than `esbuild`?

Reply:
Yes, T(ype)S(cript)config.json. While `esbuild` _supports_ transpiling typescript to javascript, it _does not_ do any sort of type checking etc., and it also will _only_ produce a bundled file. So we still need `tsc` around to type-check the sources and transpile the test code.